src/Microsoft.AspNet.Authentication.OAuth/OAuthHandler.cs

```diff
@@ -89,7 +88,8 @@ public OAuthHandler(HttpClient backchannel)
                 var value = query["error"];
                 if (!StringValues.IsNullOrEmpty(value))
                 {
-                    Logger.LogVerbose("Remote server returned an error: " + Request.QueryString);
+                    Logger.LogWarning("Remote server returned an error: " + value);
```

Comment:
Should we try to enable better control flow at this time as part of these changes?  This behavior was exactly what I ran into as well (and why I moved it to a dedicated method)